From fb389c1a04e1bcb96dd73c8ed8aad888a72037af Mon Sep 17 00:00:00 2001
From: Tristan Matthews <tristan.matthews@savoirfairelinux.com>
Date: Tue, 28 Oct 2014 17:23:35 -0400
Subject: [PATCH] datetime: fix const inconsistency

This broke the win32 build as the header prototype didn't match the
implementation.
---
 corelib/datetime.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/corelib/datetime.cpp b/corelib/datetime.cpp
index 0e2c3c3..e13fb51 100644
--- a/corelib/datetime.cpp
+++ b/corelib/datetime.cpp
@@ -89,7 +89,7 @@ void DateTime::release(tm_t *dt)
 #else
 static mutex_t lockflag;
 
-tm_t *DateTime::local(time_t *now)
+tm_t *DateTime::local(const time_t *now)
 {
     tm_t *dt;
     time_t tmp;
@@ -107,7 +107,7 @@ tm_t *DateTime::local(time_t *now)
     return NULL;
 }
 
-tm_t *DateTime::gmt(time_t *now)
+tm_t *DateTime::gmt(const time_t *now)
 {
     tm_t *dt;
     time_t tmp;
